app/services/sufia/user_stat_importer.rb

```diff
@@ -84,13 +89,19 @@ def file_ids_for_user(user)
         ids
       end
 
-      # For each date, add the view and download counts for this
-      # file to the view & download sub-totals for that day.
-      # The resulting hash will look something like this:
-      # {"2014-11-30 00:00:00 UTC" => {:views=>2, :downloads=>5},
-      #  "2014-12-01 00:00:00 UTC" => {:views=>4, :downloads=>4}}
-      def tally_results(file_stats, stat_name, total_stats)
-        file_stats.each do |stats|
+      def work_ids_for_user(user)
```

Comment:
The reason I'm suggesting this here is it may allow us to avoid having a ClassLength exception in rubocop_todo.yml